.github/workflows/post_release_version_bump.yml

```diff
@@ -0,0 +1,41 @@
+name: Automated Release Version Bumps
+
+on:
+  release:          
+    types: [published]   # run on release! 
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "post_release_version_bump"
+  post_release_version_bump:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+          
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pennylane
+
+      - name: Run version bump
+        run: |
+          python vb_script.py --version_path "./_version.py" --changelog_path "./CHANGELOG.md" --post_release 
+          
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
```

Comment:
I wonder if it is possible to auto-merge a PR if all the checks pass 🤔 although, I guess safer to have someone review it!